cylc/rose/rose.py

```diff
@@ -76,6 +105,35 @@ def get_rose_vars(dir_=None, opts=None):
                 ]
             )
 
+    # Add the entire config to ROSE_SUITE_VARIABLES to allow for programatic
+    # access.
+    from ast import literal_eval
+    for section in ['jinja2:suite.rc', 'empy:suite.rc']:
+        if config[section] is not None:
+            for key, value in config[section].items():
+                try:
+                    config[section][key] = literal_eval(value)
+                except (ValueError, SyntaxError):
+                    # An error at this point most likely results in
+                    # attempting to literal_eval('arbitary string').
+                    pass
```

Comment:
Should raise this error rather than silently dropping the variable definition.

Reply:
We don't drop it. We just don't evaluate it, so that you get the variable as a string.

Reply:
Ah, should still raise an error here though as this will likely be a backdoor for hard to debug errors e.g:

```
[env]
FOO=true

[jinja2]
# whoops that should be True
BAR=true
```

Reply:
I think that I've fixed this - take a look.

Reply:
Still says `except: pass`?

Reply:
Dunno - but it's not there now...